static/js/browse.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @param mediaEntry
+ */
```

Comment:
This is incomplete JavaDoc, so not adding any value.  This information can be read from the method itself.